presto-iceberg/src/main/java/com/facebook/presto/iceberg/optimizer/IcebergEqualityDeleteAsJoin.java

```diff
@@ -316,7 +322,7 @@ private TableScanNode createDeletesTableScan(ImmutableMap<VariableReferenceExpre
                             tableName.getSnapshotId(),
                             Optional.empty()),
                     icebergTableHandle.isSnapshotSpecified(),
-                    icebergTableHandle.getPredicate(),
+                    TupleDomain.all(),
```

Comment:
What's the reason for this change? Will the query result be wrong if the table is filtered on the predicate?

Reply:
As the predicate is useless and would be dropped subsequently, but the native worker need this value to be not null, so for safety I just set it to `ALL`.